app/workers/repository_maintenance_stat_worker.rb

```diff
@@ -0,0 +1,25 @@
+class RepositoryMaintenanceStatWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :repo_maintenance_stat, lock: :until_and_while_executing, unique_across_queues: true, retry: 3, on_conflict: :log
```

Comment:
Probably worth a comment about why the various options are chosen here

Reply:
I added a comment to describe the purpose of the settings and what I expect the behavior to be. I can also describe each setting and its expected behavior to if that makes sense.